controllers/users.controller.js

```diff
@@ -4,21 +4,21 @@ const User = require('../models/user.model');
 const db = require('../db');
 const UserSerializer = require('../utils/user-serializer');
 
-module.exports.handleAuthorizeUser = async (ctx, next) => {
+module.exports.AuthorizeUser = async (ctx, next) => {
   const user = await getDatabaseUser(ctx.state.accessToken);
   if (user.error) ctx.throw(user.code, user.error);
   ctx.status = 200;
   ctx.body = UserSerializer.serializeWithToken(user);
 };
 
-module.exports.handleUserGet = async (ctx, next) => {
+module.exports.UserGet = async (ctx, next) => {
```

Comment:
...same. However since this one is swapped (noun, verb) vs (verb, noun) I would suggest swapping them. Typically functions start with an action do make it clear that it does something.